en_us/quests/control_repository.md

```diff
@@ -146,7 +146,7 @@ Copy the site-specific `cowsay`, `pasture`, `motd`, `user_accounts`, `role`,
 and `profile` Puppet modules into the your control repository's `site`
 directory.
 
-    cp -r {pasture,motd,user_accounts,role,profile} /root/control-repo/site/
+    cp -r {cowsay,pasture,motd,user_accounts,role,profile} /root/control-repo/site/
```

Comment:
Would you mind making this same change in the `ja_jp/quests` version of the file too?

Reply:
Oh, I missed that one, let me check it out. Thanks! :)

Reply:
@greglarkin, as far as I can see, that command is also mentioned in tests/control_repository_spec.rb, should I update it there as well?

Reply:
Yes, it looks correct to me to add that directory to that test and also check for its existence just below the copy command.

Reply:
Thanks, I'll push that change as well.

Reply:
Pushed, looks OK to me.

Reply:
👍